MAUI/Carousel-View/LoadMore.md

```diff
@@ -0,0 +1,517 @@
+
+namespace CarouselSample
+{
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+            this.BindingContext = new CarouselViewModel();
+            SfCarousel carousel = new SfCarousel()
+            {
+                ItemHeight = 200,
+                ItemWidth = 200,
+                ItemSpacing = 2,
+                AllowLoadMore = true,
+                LoadMoreItemsCount = 2,
+                ViewMode = ViewMode.Linear
+            };
+
+            carousel.ItemTemplate = new DataTemplate(() =>
+            {
+                Image image = new Image();
+                image.SetBinding(Image.SourceProperty, "Image");
+                image.Aspect = Aspect.AspectFit;
+                return image;
+            });
+
+            carousel.SetBinding(SfCarousel.ItemsSourceProperty, "ImageCollection");
+
+            Grid grid = new Grid()
+            {
+                BackgroundColor = Color.White
+            };
+
+            Label label = new Label()
+            {
+                Text = "Load More...",
+                FontSize = 14,
+                TextColor = Color.Black,
+                FontAttributes = FontAttributes.Bold,
+                HorizontalOptions = LayoutOptions.Center,
+                VerticalOptions = LayoutOptions.Center,
+                HorizontalTextAlignment = TextAlignment.Center,
+                VerticalTextAlignment = TextAlignment.Center
+            };
+
+            grid.Children.Add(label);
+
+            carousel.LoadMoreView = grid;
+
+            this.Content = carousel;
+        }
+    }
+
+    public class CarouselModel
+    {
+        public CarouselModel(string imageString)
+        {
+            Image = imageString;
+        }
+        private string _image;
+
+        public string Image
+        {
+            get { return _image; }
+            set { _image = value; }
+        }
+    }
+
+    public class CarouselViewModel
+    {
+        public CarouselViewModel()
+        {
+            ImageCollection.Add(new CarouselModel("cart.png"));
+            ImageCollection.Add(new CarouselModel("cart.png"));
+            ImageCollection.Add(new CarouselModel("cart.png"));
+            ImageCollection.Add(new CarouselModel("cart.png"));
+            ImageCollection.Add(new CarouselModel("cart.png"));
+        }
+
+        private List<CarouselModel> imageCollection = new List<CarouselModel>();
+        public List<CarouselModel> ImageCollection
+        {
+            get { return imageCollection; }
+            set { imageCollection = value; }
+        }
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Load more](images/LoadMore.png)
+
+You can find the complete Load More sample from this `link.`
+
+## LoadMore method
+
+To load more items programmatically, `LoadMore` method can be called which loads the items dynamically to the UI based on the `LoadMoreItemsCount` API value.
```

Comment:
To load more items programmatically, the `LoadMore` method can be called, which loads the items dynamically to the UI based on the `LoadMoreItemsCount` API value.

Reply:
Addressed the changes.